app/store/reducers/todos.js

```diff
@@ -1,8 +1,8 @@
-import uuidv4 from 'uuid/v4';
-import {createSlice} from 'redux-starter-kit';
+import {v4 as uuidv4} from 'uuid';
```

Comment:
Importing directly causes us to have native tree-shaking, but with new destructuring importing we should analyze our elegant stack to understand do we have tree-shaking or not.

The proper way to import partials is to import them directly just like below:

```
import uuidv4 from 'uuid/v4';
```

Reply:
you're right, I just copied from the package docs
shall we keep it as it's on docs or change it as you said?

Reply:
Amer jan seems.this package is not ready this way and we have to use it as it said in its docs:
`import {v4 as uuidv4} from 'uuid';`